src/runtime/hexagon_host.cpp

```diff
@@ -918,6 +921,26 @@ WEAK void halide_hexagon_power_hvx_off_as_destructor(void *user_context, void *
     halide_hexagon_power_hvx_off(user_context);
 }
 
+WEAK int halide_hexagon_set_thread_params(void *user_context, int priority, int stack_size) {
```

Comment:
As discussed offline just now, renaming this to ``halide_hexagon_set_remote_thread_params``  makes more sense?

Reply:
Done. Thanks